site/components/src/main/java/uk/nhs/digital/intranet/model/Person.java

```diff
@@ -0,0 +1,77 @@
+package uk.nhs.digital.intranet.model;
+
+import uk.nhs.digital.intranet.enums.SearchResultType;
+
+import java.util.List;
+
+public class Person {
+
+    private String id;
+    private String displayName;
+    private String email;
+    private String jobRole;
+    private String department;
+    private String officeLocation;
+    private String phoneNumber;
+    private List<String> businessPhones;
+    private String photo;
+
+    public Person(String id, String displayName, String department) {
+        this.id = id;
+        this.displayName = displayName;
+        this.department = department;
+    }
+
+    public Person(String id, String displayName, String email, String jobRole, String department,
+                  String officeLocation, String phoneNumber, List<String> businessPhones, String photo) {
+        this.id = id;
+        this.displayName = displayName;
+        this.email = email;
```

Comment:
A small style improvement would be to call `this(id, displayName, email);`.

Reply:
In that case, I would have to call the setters for the rest of the fields. Is that really an improvement?

Reply:
Far point.